### PR TITLE
cleanup deprecation warnings

### PR DESCRIPTION
**Pull request by ecoologic**
I had some problems running the tests, which I solved looking at the other Gemfiles and adding the missing dependencies.

I'm not sure how I was supposed to use the Gemfile the way it was because Rails was not defined as a dependency anywhere.

Happy to change it if I get to grasp it.

At least I can run the specs now (:

### CODE REVIEW

File: spec/scenario/config/initializers/secret_token.rb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 Rails.application.config.secret_token = 'ea942c41850d502f2c8283e26bdc57829f471bb18224ddff0a192c4f32cdf6cb5aa0d82b3a7a7adbeb640c4b06f3aa1cd5f098162d8240f669b39d6b49680571'
-Rails.application.config.session_store :cookie_store, :key => "_my_scenario"
\ No newline at end of file
+Rails.application.config.session_store :cookie_store, key: "_my_scenario"
+Scenario::Application.config.secret_key_base = 'ea942c41850d502f3c8283e26bdc57829f471bb18224ddff0a192c4f32cdf6cb5aa0d82b3a7a7adbeb640c4b06f3aa1cd5f098162d8240f669b39d6b49680574'
```
Comment (ecoologic) [inline spec/scenario/config/initializers/secret_token.rb]: also needed to solve deprecation warning

**Comment by nbudin** [comment]
Hi, thanks for that!  It's been so long since I set this up that I'm actually not sure what the deal with the Gemfile at the root is either, but it seems to be working this way so I don't see any problem with these changes.

**Comment by ecoologic** [comment]
@nbudin Thanks for that.  I have another one, you can see the diff with your previous master here:

https://github.com/ecoologic/devise_openid_authenticatable/pull/1/files

I would love a hint to fix this build:

https://travis-ci.org/ecoologic/devise_openid_authenticatable/jobs/151983113

Basically, devise 3.4 (changelog: https://github.com/plataformatec/devise/blob/3-stable/CHANGELOG.md#340---2014-10-03) introduces some changes that break our build.

What I haven't understood so far is:  Should I fix the spec or should I fix the gem itself?

I don't think `remember_token` is ever assigned.

Any suggestion is appreciated.

**Comment by nbudin** [comment]
I'd still like to test somehow that this doesn't break rememberable, so testing that the user is remembered seems like a good thing.  But if this is no longer how Devise does it, then we'll have to fix the spec.  (We can't fix the gem because the gem has really nothing to do with it.)

Looking at that changelog it's not obvious to me how Devise broke remember_token functionality, and it does seem to still be present in its codebase.  Could you say more about the breakage?

**Comment by ecoologic** [comment]
I 100% agree with you, we need to keep the spec.  I'm investigating the bug, but damn, it's fairly new territory.  I'll get there and report back.

Thanks so much for your latest update, it's a very important confirmation.

I can't say much more so far, the only hint I have is that `user.save` is called twice, and the second time the token is saved.

Hopefully, I'm still on top of it.

Thanks

**Pull request merged**